src/main/java/net/javadiscord/javabot/data/config/guild/ModerationConfig.java

```diff
@@ -21,6 +21,8 @@ public class ModerationConfig extends GuildConfigItem {
 	private long adminRoleId;
 	private long expertRoleId;
 
+	private long minActiveHoursToSendMedia;
```

Comment:
Remove this empy line
```suggestion
	private long expertRoleId;
	private long minActiveHoursToSendMedia;
```